calico_containers/integrations/kubernetes/calico_kubernetes.py

```diff
@@ -260,14 +262,26 @@ def _generate_rules(self):
         inbound_rules = [
             {
                 'action': 'allow',
-            },
+            }
         ]
 
         outbound_rules = [
             {
                 'action': 'allow',
-            },
+            }
         ]
+
+        print('Getting Policy Rules from Annotation of pod %s' % pod)
+
+        annotations = self._get_metadata(pod, 'annotations')
+        namespace = self._get_metadata(pod, 'namespace')
+
+        if 'allowFrom' in annotations.keys():
+            inbound_rules = load_json(annotations['allowFrom'])
```

Comment:
I wouldn't load directly to inbound_rules.  There might be problems with the loaded rules.  We should load the rules into their own list, translate each one, and if it succeeds translation then add it to the list.

The other thing to think about is what do we do if a rule is not valid? Do we carry on and try to do our best?  Its probably not a security concern (yet), since by default traffic will be dropped, and each rule is a whitelist entry.  But we should be logging appropriate, human readable errors when an allowFrom rule is invalid.